Library/Homebrew/os/mac.rb

```diff
@@ -3,7 +3,7 @@
 require "os/mac/version"
 require "os/mac/xcode"
 require "os/mac/xquartz"
-require "os/mac/pathname"
+require "extend/os/mac/extend/pathname"
```

Comment:
Feels weird to require deeply here.

Reply:
My opinion…
`os/mac.rb` should not require `extend/os/mac/extend/pathname.rb`
`extend/pathname.rb` should require `extend/os/pathname.rb`